molecule/pdmysql/tests/test_common_pxb_pt.py

```diff
@@ -25,7 +25,8 @@
 
 PXB_VERSION = os.getenv("PXB_VERSION")
 PT_VERSION = os.getenv("PT_VERSION")
-
+REPO = os.environ.get("REPO")
+TO_REPO = os.environ.get("TO_REPO")
```

Comment:
Instead of TO_REPO you can mark the test as for install only.
[9:42](https://perconacorp.slack.com/archives/D027PCR2WQZ/p1683009756920619)
https://github.com/Percona-QA/package-testing/blob/71e72cb5ddaa102076c04f3a9f36829546c66c4a/molecule/pdmysql/tests/test_pdps_ps.py#L169

Reply:
Done : https://ps80.cd.percona.com/view/PDPS/job/pdps-upgrade/84/